website/docs/docs/build/conversion-metrics.md

```diff
@@ -0,0 +1,352 @@
+          name: buys
+          fill_nulls_with: 0
+        entity: user
+        window: 1 week
+```
+
+</TabItem>
+
+<TabItem value="constproperty" label="Set constant property">
+
+*Refer to [Amplitude's blog posts on constant properties](https://amplitude.com/blog/holding-constant) to learn about this concept.*
+
+You can add a constant property to a conversion metric to count only those conversions where a specific dimension or entity matches in both the base and conversion events. 
+
+For example, if you're at an e-commerce company and want to answer the following questions:
+- _How often did visitors convert from `View Item Details` to `Complete Purchase` with the same product in each step?_<br />
+  - What makes this question tricky to answer is users could have completed these two conversion milestones across many products. For example, viewed a pair of shoes, then a T-shirt,  and eventually checked out with a bow tie. This would still count as a conversion, even though the conversion event only happened for the bow tie.
+
+Back to our initial questions, we want to see how many customers viewed an item detail page and then completed a purchase for the _same_ product.
+
+In this case, we would want to set `product_id` as the constant property. We would specify this in the configs as follows:
+
+```yaml
+- name: view_item_detail_to_purchase_with_same_item
+  description: "Conversion rate for users who viewed the item detail page and purchased the item"
+  type: Conversion
+  label: View Item Detail > Purchase
+  type_params:
+    conversion_type_params:
+      calculation: conversions
+      base_measure: view_item_detail
+      conversion_measure: purchase
+      entity: user
+      window: 1 week
+      constant_properties:
+        - base_property: product
+          conversion_property: product
+```
+
+We will add an additional condition to the join to make sure the constant property is the same across conversion.
+
+```sql
+select distinct
+  first_value(v.ds) over (partition by buy_source.ds, buy_source.user_id, buy_source.session_id order by v.ds desc rows between unbounded preceding and unbounded following) as ds,
+  first_value(v.user_id) over (partition by buy_source.ds, buy_source.user_id, buy_source.session_id order by v.ds desc rows between unbounded preceding and unbounded following) as user_id,
+  first_value(v.referrer_id) over (partition by buy_source.ds, buy_source.user_id, buy_source.session_id order by v.ds desc rows between unbounded preceding and unbounded following) as referrer_id,
+  buy_source.uuid,
+  1 as buys
+from {{ source_schema }}.fct_view_item_details v
+inner join
+  (
+    select *, {{ generate_random_uuid() }} as uuid from {{ source_schema }}.fct_purchases
+  ) buy_source
+on
+  v.user_id = buy_source.user_id
+  and v.ds <= buy_source.ds
+  and v.ds > buy_source.ds - interval '7 day'
+  and buy_source.product_id = v.product_id --Joining on the constant property product_id
+
+```
+
+</TabItem>
+</Tabs>
```

Comment:
this is consistent with all other metric types so it might be fine for now, until we add more functions. I'm going to cc @Jstein77 in case we can add more use cases or functions (like filter)?